app/assets/stylesheets/layout.scss

```diff
@@ -97,6 +97,12 @@ body {
 }
 
 //pagination
+
+.pagination {
```

Comment:
Duplicate rules ?

Reply:
@tachyons sir firstly i put overflow-x:scroll in media query to show scrollbar only in small devices  , but as suggested by @gr455 sir overflow-x : auto now i have removed the media query because auto will automatically show scrollbar only when needed✌ .
Now i have commited the new changes 😃

Reply:
@Palaksharma23 He means you have made a serpeate style for pagination again although it exists already [here](https://github.com/CircuitVerse/CircuitVerse/pull/2613/files/e8424ee813c425178452f5d65b5532a5bc578475#diff-4b04ba88a773a7736d1e8a5bcb978cf7faa74c935b4389bcd9d962407c9a8149R132)

Reply:
yes @gr455 @tachyons actually first it was under media query which was producing issue  , now i have defined it as general and now it is working fine ✌